### PR TITLE
Http handle cors

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-scale-tests
run skydive-go-fmt
run skydive-cdd-overview-tests

**Comment by safchain** [comment]
run skydive-scale-tests
run skydive-cdd-overview-tests

**Comment by safchain** [comment]
run skydive-compile-tests
run skydive-scale-tests

File: http/server.go [hunk 1]
```diff
@@ -119,7 +119,11 @@ func (s *Server) Serve() {
 	defer s.wg.Done()
 	s.wg.Add(1)
 
-	s.Handler = handlers.CompressHandler(s.Router)
+	headersOk := handlers.AllowedHeaders([]string{"X-Requested-With", "Content-Type", "Authorization", "X-Auth-Token"})
+	originsOk := handlers.AllowedOrigins([]string{"*"})
+	methodsOk := handlers.AllowedMethods([]string{"GET", "HEAD", "POST", "PUT", "OPTIONS"})
```
[the remark this answers is not in the document]
Comment (nplanel) [reply http/server.go]: following Mozilla recommendation : https://developer.mozilla.org/fr/docs/Web/HTTP/CORS
Only request GET, HEAD, POST are part of http CORS

[the remark this answers is not in the document]
Comment (safchain) [reply http/server.go]: pre-flight requests use option

[the remark this answers is not in the document]
Comment (safchain) [reply http/server.go]: can be validated using the UI authentication

**Pull request merged**